signer/src/stacks/api.rs

```diff
@@ -902,11 +899,21 @@ impl StacksInteract for StacksClient {
     where
         T: AsTxPayload + Send + Sync,
     {
+        // Consensus serialize the transaction payload to bytes. This is the
+        // method that the stacks node uses to determine transaction size when
+        // verifying admittance into the mempool.
+        let payload_bytes = payload.tx_payload().serialize_to_vec();
+
+        // In Stacks core, the minimum fee is 1 mSTX per byte, so we double
+        // that here as a precaution and cap it at our maximum fee.
+        let default_min_fee =
+            (payload_bytes.len() as u64 * TX_FEE_PAYLOAD_SIZE_MULTIPLIER).min(MAX_TX_FEE);
```

Comment:
Oh we still need to update this to use the new `get_full_tx_size`

Reply:
Yeah I had just pushed changes + test for `get_full_tx_size`, will fix up the above now